selenium/integration/pipeline.hierarchy.spec.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { By, Key } from 'selenium-webdriver';
+import { loginIfRequired, buildChromeDriver, dataTestId, getGenericEndpoint } from '../support/utils';
+import { PIPELINE_STUDIO_URL } from '../support/constants';
+import Selenium from '../support/commands';
+
+let driver;
+let selenium;
+
+describe.skip('Hierarchy Widgets', () => {
```

Comment:
This test is currently being skipped. I believe that's due to the `CreateRecord` plugin no longer existing in `2.10.0-SNAPSHOT` plugins (or possibly prior versions as well). However, I was able to test all the commands I added using the `File` plugin. It's an important test as it defines the `commands` structure for Selenium.